code/tests/test_vector_stores.py

```diff
@@ -28,15 +28,15 @@ def test_update_db_with_crawled_data(input_db, crawl_2, request):
     data_add, ids_update_last_seen, ids_del = compare_crawled_data_with_db(db, crawl_2)
 
     assert len(data_add) == 2, "Expected 2 objects to add"
-    assert data_add[0].metadata["id"] == "id4#4c"
-    assert data_add[1].metadata["id"] == "id5#5"
+    assert data_add[0].metadata["id"] == "00000000-0000-0000-0000-00000000004c"
+    assert data_add[1].metadata["id"] == "00000000-0000-0000-0000-000000000005"
 
     assert len(ids_update_last_seen) == 1, "Expected 1 object to update"
-    assert "id3#3" in ids_update_last_seen, "Expected id3#3 to be updated"
+    assert "00000000-0000-0000-0000-000000000003" in ids_update_last_seen, "Expected 00000000-0000-0000-0000-000000000003 to be updated"
```

Comment:
Qdrant only allows UUIDs and unsigned int as point IDs.So the replacement.
Should work well with the other DBs too.
https://qdrant.tech/documentation/concepts/points/#point-ids

Reply:
> Qdrant only allows UUIDs and unsigned int as point IDs.So the replacement. Should work well with the other DBs too. https://qdrant.tech/documentation/concepts/points/#point-ids

This looks quite ugly :). I saw that qdrant also accepts int as ids. But the other two dbs are not supporting that.
What might be good to create variables in conftest.py

```
ID4c = "00000000-0000-0000-0000-00000000004c"
d4c = Document(
    page_content="Changed->add-new", metadata={"item_id": "id4", "id": ID4c, "checksum": "0", "last_seen_at": 2}
)
```

Then, in the test, we can use:
```
assert data_add[0].metadata["id"] == ID4c
```

Can you please change it this way?